settings/detection.py

```diff
@@ -0,0 +1,64 @@
+from registry import DECISION_STRATEGIES, DETECTION_METHODS, DETECTION_METHODS_SETTINGS
+from settings import Settings
+from utils.logging import main_logger
+
+logger = main_logger.getChild("settings.detection")
+
+
+class DetectionSettings(Settings):
```

Comment:
docs

Reply:
No, actually. I forgot to add it and was confused why nothing was saving.